artemis-web/src/test/java/org/apache/activemq/cli/test/WebServerComponentTest.java

```diff
@@ -880,6 +882,48 @@ public void testServerDeterministicWebappName() throws Exception {
       assertFalse(webServerComponent.isStarted());
    }
 
+   @Test
+   public void testIsolationOfSessions() throws Exception {
+      BindingDTO bindingDTO1 = new BindingDTO();
+      bindingDTO1.uri = "http://localhost:0";
+      bindingDTO1.apps = new ArrayList<>();
+      bindingDTO1.apps.add(createSimpleAppDTO("app1"));
+      bindingDTO1.apps.add(createSimpleAppDTO("app2"));
+
+      BindingDTO bindingDTO2 = new BindingDTO();
+      bindingDTO2.uri = "http://localhost:0";
+      bindingDTO2.apps = new ArrayList<>();
+      bindingDTO2.apps.add(createSimpleAppDTO("app1"));
+      bindingDTO2.apps.add(createSimpleAppDTO("app2"));
+
+      WebServerDTO webServerDTO = new WebServerDTO();
+      webServerDTO.setBindings(Arrays.asList(bindingDTO1, bindingDTO2));
```

Comment:
This is a nit-pick so feel free to ignore, but `List.of` is a bit more readable than `Arrays.asList` in my opinion.